app/views/layouts/application.html.erb

```diff
@@ -13,17 +13,25 @@
 
     <script src="https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.0.1/dist/alpine.js" defer></script>
 
-    <% if Rails.env == 'production' %>
-      <!-- Global site tag (gtag.js) - Google Analytics -->
-      <script async src="https://www.googletagmanager.com/gtag/js?id=UA-162054776-1"></script>
+      <%# Always load GA, for consistency, but use different accounts for development vs. production %>
+      <script async src="https://www.googletagmanager.com/gtag/js?id=<%= google_analytics_id %>"></script>
       <script>
         window.dataLayer = window.dataLayer || [];
         function gtag(){dataLayer.push(arguments);}
         gtag('js', new Date());
 
-        gtag('config', 'UA-162054776-1');
+        gtag('config', '<%= google_analytics_id %>');
+
+        <% action = pop_event_to_track %>
```

Comment:
Can you rename this something other than action?  I can see this being confused with rails's action.

Reply:
Yep will do. Will move the controller method to the helper too, thx